tools/test-shards.ts

```diff
@@ -1,5 +1,6 @@
 import crypto from 'node:crypto';
 import { minimatch } from 'minimatch';
+import { getEnv } from '../lib/util/env';
```

Comment:
no, revert this. this should use real env and isn't part of renovate package / distribution.